package.json

```diff
@@ -20,7 +20,8 @@
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "engines": {
-    "node": ">=10"
+    "node": ">=10",
+    "npm": ">=6"
```

Comment:
This engines are for consuming code. I don't think this library requires npm.

Reply:
It's also checked I think

Reply:
What do you mean by "it's checked"?

Reply:
When you run `npm install`, the version is checked.
Just like you have when using `yarn`.

Reply:
Yeah but it shouldn't. I can install this library just as well with yarn or pnpm.

Reply:
The version is only checked when using `npm`

Reply:
> You can also use the “engines” field to specify which versions of npm are capable of properly installing your program.

-- https://docs.npmjs.com/files/package.json#engines

Why do we need npm 6 to install `@testing-library/dom`?

Reply:
It's not necessary, but just setting it as a sensible baseline
Most people are already using `npm@6`

Reply:
Any code that is not necessary should be removed.

Reply:
Yeah, I think we should probably just remove the npm engine. It's really not necessary.

Reply:
I'll remove it then 🙂